src/main/java/constants/SelectionOption.java

```diff
@@ -0,0 +1,26 @@
+package constants;
+
+import java.util.Arrays;
+
+public enum SelectionOption {
+    ADD_ATTENDANCE("1"),
+    EDIT_ATTENDANCE("2"),
+    GET_ATTENDANCE_HISTORY("3"),
+    CHECK_ABSENCE_USERS("4"),
+    QUIT("Q");
+    private final String option;
```

Comment:
줄바꿈을 통해서 상태와 enum 요소를 구분하기 쉽게해주면 가독성이 좋아질 것 같아요. 😃 

```suggestion
    QUIT("Q");
    
    private final String option;
```

Reply:
지금보니 로키 말대로 공백 컨벤션을 잘 지키지 못한거 같네요! 좋은 지적 감사합니다.😊